VultisigApp/VultisigApp/Chains/common.swift

```diff
@@ -68,6 +68,9 @@ struct SignatureProvider {
             switch sigResult {
             case .success(let sigData):
                 logger.info("successfully get signature")
+                
+                print("successfully get signature")
```

Comment:
clean up here?

Reply:
Oh, I will )

Reply:
I've cleaned in this other one. https://github.com/vultisig/vultisig-ios/pull/1659